app/mailers/host_mailer.rb

```diff
@@ -39,7 +39,8 @@ def summary(options = {})
   def error_state(report)
     report = Report.find(report)
     host = report.host
-    owners = host.owner.recipients_for(:puppet_error_state)
+    owners = host.owner.recipients_for(:puppet_error_state) if host.owner.present?
+    owners ||= []
     raise ::Foreman::Exception.new(N_("unable to find recipients")) if owners.empty?
```

Comment:
line 44 will never raise now, i believe the indent was simply to render nothing if there are no owners?